examples/threads.rs

```diff
@@ -48,36 +50,49 @@ async fn main() -> Result<()> {
         }
     };
 
-    let _address = account.generate_addresses(10, None).await?;
-    for ad in _address {
-        println!("{}", ad.address().to_bech32());
-    }
+    // One address gets generated during account creation
+    let address = account.list_addresses().await?[0].address().clone();
+    println!("{}", address.to_bech32());
+
     let balance = account.sync(None).await?;
     println!("Balance: {:?}", balance);
 
+    if balance.available == 0 {
+        panic!("Account has no available balance");
+    }
+
     for _ in 0..1000 {
         let mut threads = Vec::new();
-        for n in 0..5 {
+        for n in 0..10 {
             let account_ = account.clone();
+            let address_ = *address.as_ref();
+
             threads.push(async move {
                 tokio::spawn(async move {
                     // send transaction
                     let outputs = vec![
                         BasicOutputBuilder::new_with_amount(1_000_000)?
-                            .add_unlock_condition(UnlockCondition::Address(AddressUnlockCondition::new(
-                                Address::try_from_bech32(
-                                    "rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu",
-                                )?
-                                .1,
-                            )))
-                            .finish_output()?,
+                            .add_unlock_condition(UnlockCondition::Address(AddressUnlockCondition::new(address_)))
+                            .finish_output()?;
+                        // amount of outputs in the transaction (one additional output might be added for the remaining amount)
+                        1
```

Comment:
Any reason we explicit this ?

Reply:
So it's easy to change to another number, like 100 for example

Reply:
Fair enough, was curious